### PR TITLE
Tabs refactor

### PR DESCRIPTION
**Pull request by wa-rren-dev**

### CODE REVIEW

File: packages/components/tabs/tabs.js [hunk 1]
```diff
@@ -1,313 +1,303 @@
...
+      this.setupResponsiveChecks();
+    } else {
+      this.setup();
+    }
   }
-};
 
-Tabs.prototype.setupResponsiveChecks = function setupResponsiveChecks() {
-  this.mql = window.matchMedia('(min-width: 40.0625em)');
-  this.mql.addListener(this.checkMode.bind(this));
```
Comment (wa-rren-dev) [inline packages/components/tabs/tabs.js:33]: IDE was showing addListener as deprecated

File: packages/components/tabs/tabs.js [hunk 2]
```diff
@@ -1,313 +1,309 @@
...
-  this.checkMode();
-};
+  setupResponsiveChecks() {
+    // $mq-breakpoints: (
+    // mobile: 320px,
+    // tablet: 641px,
+    // desktop: 769px,
+    // large - desktop: 990px
+    // );
+    this.mql = window.matchMedia('(min-width: 641px)');
```
Comment (wa-rren-dev) [inline packages/components/tabs/tabs.js:33]: Have updated here to match our current tablet size

File: packages/components/tabs/tabs.js (hunk 1 above)
Comment (andymantell) [reply packages/components/tabs/tabs.js:33]: yeah that's fair, good spot

File: packages/components/tabs/tabs.js (hunk 2 above)
Comment (andymantell) [reply packages/components/tabs/tabs.js:33]: 👍🏻

File: packages/components/tabs/tabs.js [hunk 3]
```diff
@@ -1,313 +1,309 @@
-// import "../../vendor/polyfills/Function/prototype/bind";
```
Comment (andymantell) [inline packages/components/tabs/tabs.js:1]: @wa-rren-dev This diff is super noisy - could you summarise what things you have changed here?

**Review by andymantell — APPROVED** [review]

**Pull request merged**